episodes/example.Rmd

```diff
@@ -112,6 +112,15 @@ accessibility purposes'}`
 
 ![You belong in The Carpentries!](https://raw.githubusercontent.com/carpentries/logo/master/Badge_Carpentries.svg){alt='Blue Carpentries hex person logo with no text.'}
 
+Additional attributes can be specified for the image alongside the alternative etxt description
+in the `{}`.
+For example, to add some space around the image, you could apply some padding:
+
+`![You belong in The Carpentries!](https://raw.githubusercontent.com/carpentries/logo/master/Badge_Carpentries.svg){alt='Blue Carpentries hex person logo with no text.' padding=10px}`
+
+![You belong in The Carpentries!](https://raw.githubusercontent.com/carpentries/logo/master/Badge_Carpentries.svg){alt='Blue Carpentries hex person logo with no text.' padding=10px}
```

Comment:
The thing about this is that you can not add padding as an attribute here because padding is a CSS directive, which goes in the `style` attribute (e.g. `style = "padding: 25px;"`). The [attributes for images](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Img#attributes) are fairly limited. 

The method to add custom css attributes is to add a chunk embedding CSS in the document and then adding a class as shown in https://github.com/carpentries/sandpaper/pull/334.

Reply:
```suggestion
For example, to add some space around the image and create [a custom image anchor tag](#custom-carpentries-logo), you could apply some padding in the `style` attribute and add the anchor tag `#custom-image-anchor`: 

`![You belong in The Carpentries!](https://raw.githubusercontent.com/carpentries/logo/master/Badge_Carpentries.svg){#custom-carpentries-logo alt='Blue Carpentries hex person logo with no text.' style="padding: 10px;"}`

![You belong in The Carpentries!](https://raw.githubusercontent.com/carpentries/logo/master/Badge_Carpentries.svg){# custom-carpentries-logo alt='Blue Carpentries hex person logo with no text.' style='padding: 10px;'}
```

If you wanted to add more complex styling, [you can add a css code chunk](https://bookdown.org/yihui/rmarkdown-cookbook/html-css.html) to apply to a custom class.

Reply:
Sorry that I didn't test this before I opened the PR. Just checking: did you mean to write `#custom-image-anchor` above, instead of `#custom-carpentries-logo` again?

Reply:
that's what I get for making a complex example...

My question: do you think it would be better to just use `![...](...){width="25%"}` (even though it's silently converted to CSS by pandoc) and then add the link to the CSS code chunk documentation?

Reply:
also, yes, making the anchors the same would be optimal.

Reply:
Pointing people to https://bookdown.org/yihui/rmarkdown-cookbook/html-css.html for how to define custom CSS classes in their lessons will work great for lessons using Rmd source files. But, if I understand correctly how it works, it is not applicable to Markdown source files, as they are not knitted and as such the conversion from code chunk to `<style>` element will not take place.

If I have this right, the `![...](...){alt='...' style='padding=10px; border=2px;'}` will work for both kinds of lesson?